audit/projects/k8s-staging-cloud-provider-ibm/buckets/artifacts.k8s-staging-cloud-provider-ibm.appspot.com/iam.json

```diff
@@ -0,0 +1,35 @@
+{
+  "bindings": [
+    {
+      "members": [
+        "projectEditor:k8s-staging-cloud-provider-ibm",
+        "projectOwner:k8s-staging-cloud-provider-ibm"
+      ],
+      "role": "roles/storage.legacyBucketOwner"
+    },
```

Comment:
Expected, this was https://github.com/kubernetes/k8s.io/pull/2618#issuecomment-916224941